terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -64,8 +64,6 @@ module "glue_reporting_hub_job" {
     "--dpr.domain.catalog.db"                   = module.glue_data_domain_database.db_name
     "--dpr.redshift.secrets.name"               = "${local.project}-redshift-secret-${local.environment}"
     "--dpr.datamart.db.name"                    = "datamart"
-    "--spark.driver.memory"                     = local.reporting_hub_driver_mem
-    "--spark.executor.memory"                   = local.reporting_hub_executor_mem
```

Comment:
These parameters are ignored by Glue so removing them for now.
This will be later investigated in dev but not urgent for now as the current config works.